Packs/CommonWidgets/ReleaseNotes/1_1_6.md

```diff
@@ -0,0 +1,6 @@
+
+#### Scripts
+##### GetLargestInputsAndOuputsInIncidents
+- The script has been deprecated. For XSOAR versions up to 6.2.0, it is ill advised to use the script, as it can cause performance issues. For XSOAR 6.2.0 and later versions, please checkout the System Diagnostic page for an alternative.
```

Comment:
```suggestion
- The script has been deprecated. For XSOAR versions up to 6.2.0, it is not advised to use the scrip as it can cause performance issues. For XSOAR 6.2.0 and later versions, please refer to the System Diagnostic page as an alternative.
```